gapic/src/main/com/google/gapid/widgets/Widgets.java

```diff
@@ -342,6 +343,18 @@ public static Spinner createSpinner(Composite parent, int value, int min, int ma
     result.setMinimum(min);
     result.setMaximum(max);
     result.setSelection(value);
+
+    if (OS.isMac) {
+      result.addListener(SWT.KeyUp, e -> {
+        if ((e.stateMask & (SWT.CONTROL | SWT.COMMAND)) != 0) {
+          switch (e.keyCode) {
+            case 'c': result.copy(); break;
```

Comment:
We have code like this somewhere else that I wrote for the shader editor - maybe we should unify this so we don't have several places to maintain?

Reply:
I think you are referring to the undo/redo keys you added. Also, unfortunately, the copy/paste/cut methods are declared on the Spinner class, so this code can't easily be reused :disappointed: